bundler/spec/bundler/definition_spec.rb

```diff
@@ -10,34 +10,34 @@
       allow(Bundler).to receive(:ui) { double("UI", :info => "", :debug => "") }
     end
     context "when it's not possible to write to the file" do
-      subject { Bundler::Definition.new(nil, [], Bundler::SourceList.new, []) }
+      subject { Bundler::Definition.new("Gemfile.lock", [], Bundler::SourceList.new, []) }
```

Comment:
Your other caller [converts to Pathname](https://github.com/rubygems/rubygems/pull/7047/files#diff-f6ec8889d95d366c2ac4d3b18240fc23eb9486bce5e4a999991ed557ff961d7aR37) before calling, however, this is inconsistent across initializations of Definition. Should we test for receiving a Pathname too?

Reply:
I looked into this and I believe all our callers actually use `Pathname`'s (and method headers explicitly mention `Pathname` as the expected type). So what I did was to change these unit specs to also recieve `Pathname`'s.

I think this is enough but let me know if we should explicitly be covering passing strings.